src-tauri/src/main.rs

```diff
@@ -1,16 +1,89 @@
 #![cfg_attr(
-  all(not(debug_assertions), target_os = "windows"),
-  windows_subsystem = "windows"
+    all(not(debug_assertions), target_os = "windows"),
+    windows_subsystem = "windows"
 )]
 
-fn main() {
-  let context = tauri::generate_context!();
-  tauri::Builder::default()
-    .menu(if cfg!(target_os = "macos") {
-      tauri::Menu::os_default(&context.package_info().name)
+use std::sync::Mutex;
+
+use dropbox_sdk::oauth2;
+
+const APPKEY: &str = "z4c46xuhvi38jhh";
```

Comment:
@satrams is this something you can pull in as an environment variable?

Reply:
@satrams these seems relevant: https://www.thorsten-hans.com/working-with-environment-variables-in-rust/